.github/workflows/enos-release-testing-oss.yml

```diff
@@ -12,32 +12,39 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       matrix-test-group: ${{ steps.matrix-group.outputs.matrix-test-group }}
+      vault-revision: ${{ steps.get-metadata.outputs.vault-revision }}
+      vault-version: ${{ steps.get-metadata.outputs.vault-version }}
     steps:
       - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.client_payload.payload.sha }}
       - id: matrix-group
         env:
           # MATRIX_MAX_TEST_GROUPS is required to determine the randomly selected
           # test group. It should be set to the highest test_group used in the
           # enos-run-matrices.
           MATRIX_MAX_TEST_GROUPS: 2
-        run: echo "matrix-test-group=$(make ci-get-matrix-group-id)" >> $GITHUB_OUTPUT
+        run: |
+          echo "matrix-test-group=$(make ci-get-matrix-group-id)" >> $GITHUB_OUTPUT
+          echo "vault-revision=$(make ci-get-revision)" >> $GITHUB_OUTPUT
+          echo "vault-version=$(make ci-get-version)" >> $GITHUB_OUTPUT
 
   test:
-    name: Test ${{ matrix.matrix-file-name }}
+    name: Test ${{ matrix.build-artifact-name }}
     if: ${{ startsWith(github.event.client_payload.payload.branch, 'release/') }}
     needs: metadata
     uses: ./.github/workflows/enos-run.yml
     strategy:
       fail-fast: false
       matrix:
         include:
-          - matrix-file-name: enos_release_testing_oss-artifactory-oss-linux-amd64-zip
-            test-name: Linux AMD64 Zip
+          - matrix-file-name: enos_release_testing_oss-artifactory-oss-linux-amd65-zip
```

Comment:
I'm guessing that this shouldn't be `amd65`, but rather `amd64`.

Reply:
🗡️